internal/gen/types/swift/swift.go

```diff
@@ -0,0 +1,102 @@
+package swift
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/network"
+	"github.com/go-errors/errors"
+	"github.com/jackc/pgconn"
+	"github.com/jackc/pgx/v4"
+	"github.com/spf13/afero"
+	"github.com/supabase/cli/internal/utils"
+	"github.com/supabase/cli/pkg/api"
+)
+
+func Run(ctx context.Context, projectId string, dbConfig pgconn.Config, schemas []string, postgrestV9Compat bool, fsys afero.Fs, options ...func(*pgx.ConnConfig)) error {
+	originalURL := utils.ToPostgresURL(dbConfig)
+	// Add default schemas if --schema flag is not specified
+	if len(schemas) == 0 {
+		schemas = utils.RemoveDuplicates(append([]string{"public"}, utils.Config.Api.Schemas...))
+	}
+	included := strings.Join(schemas, ",")
+
+	if projectId != "" {
+		resp, err := utils.GetSupabase().V1GenerateTypescriptTypesWithResponse(ctx, projectId, &api.V1GenerateTypescriptTypesParams{
```

Comment:
TODO: change this to call `V1GenerateSwiftTypesWithResponse` once it is implemented.